sympy/ntheory/primetest.py

```diff
@@ -434,8 +434,7 @@ def is_extra_strong_lucas_prp(n):
     used in OEIS A217719, Perl's Math::Prime::Util, and the Lucas pseudoprime
     page on Wikipedia.
 
-    With these parameters, there are no counterexamples below 2^64 nor any
```

Comment:
This idea is supported in the general `isprime` where Baille-PSW is used (see https://en.wikipedia.org/wiki/Baillie%E2%80%93PSW_primality_test from whence the comment comes).